profiling.go

```diff
@@ -0,0 +1,68 @@
+package duckdb
+
+/*
+#include <duckdb.h>
+*/
+import "C"
+
+import (
+	"unsafe"
+)
+
+type ProfilingInfo struct {
+	Metrics  map[string]string
+	Children []ProfilingInfo
+}
+
+func GetProfilingInfo(driverConn any) (ProfilingInfo, error) {
```

Comment:
Would it make sense to add a comment to explain what profiling information is returned (e.g. link to the example at https://duckdb.org/docs/dev/profiling.html#setting-custom-metrics-examples) or to add a section to the README?

Reply:
Yes, good point 👍